api/package.json

```diff
@@ -12,21 +12,22 @@
     "@aws-sdk/util-dynamodb": "^3.211.0",
     "@sentry/serverless": "^7.22.0",
     "@serverless-stack/node": "^1.18.2",
-    "@ucanto/interface": "^3.0.1",
-    "@ucanto/principal": "^3.0.1",
-    "@ucanto/server": "^3.0.4",
-    "@ucanto/transport": "^3.0.2",
+    "@ucanto/client": "^4.0.2",
```

Comment:
yikes, were we missing this dep previously!?

Reply:
yeah :) felt good to find that